src/common/actions/auth-store.js

```diff
@@ -270,7 +272,13 @@ async function setShortNamespace(root, discharge, userName) {
     }
     return data;
   } else {
-    const json = await response.json();
+    const text = await response.text();
```

Comment:
Heh, same point was raised in the PR that got approved for landing here (lack of support for stacked branches in GitHub lead to this approach of C -> B -> A -> master).

Please see https://github.com/canonical-ols/build.snapcraft.io/pull/683  tl;dr: unknown, but being conservative.